pitaya-sharp/NPitaya/src/Metrics/PitayaReporter.cs

```diff
@@ -182,6 +183,24 @@ static unsafe string[] ReadLabels(IntPtr labelsPtr, UInt32 size)
             return labels;
         }
 
+        static unsafe HistogramBuckets ReadBuckets(ref BucketOpts buckets)
```

Comment:
Any reason to use `ref` here?

Reply:
Because `HistogramBuckets` is a struct and we want to pass the pointer to this value to make the stack lighter.